examples/taco/nodejs/src/index.ts

```diff
@@ -81,18 +80,6 @@ const encryptToBytes = async (messageString: string) => {
     encryptorSigner,
   );
 
```

Comment:
Are we still "checking contract compatibility" even when removing this from the Testnet examples that are run every hour?

https://github.com/nucypher/taco-web/actions/workflows/lynx.yml

Reply:
At the moment, I don't think `isAuthorized` is useful as part of the exposed `taco` API, hence the removal here. If others feel that it is useful I can put it back.

It's unclear to me how comprehensive this check is as it pertains to "checking compatibility" based on one function.  Do you have any additional insight there, perhaps I'm missing something?  Perhaps testing the `DkgCoordinatorAgent` with `lynx` contracts could be a better solution to CI checking compatibility...maybe - not sure what that entails or if it is feasible?

Additionally, `Coordinator.isEncryptionAuthorized()` which is called by `isAuthorized()`, is actually deprecated in the latest Coordinator contract, and will be removed down the road. Instead, the underlying acccessController for the ritual Id should be called directly for that check.

Reply:
No, I don't have a clear idea about this check beyond that we can check that this specific method on the Coordinator contract has not been modified. 

So yes, I agree we can delete this.

My concern came after I read the commented lines in the code talking about "checking contract compatibility": I was wondering if I'm missing something here.